analyze-question-sentiment.go

```diff
@@ -131,6 +132,26 @@ func analyzeDir(baseDir string, tag string, fromDate time.Time, toDate time.Time
 	return tr
 }
 
+// Discover the names of the subfolders inside dir (non-recursively).
+func readFolderNames(dirpath string) ([]string, error) {
```

Comment:
Since this is just a "script" I'd recommend simplifying by doing error checking close to source and not propagating errors. It's not like `main` is going to do something intelligent with `err` - it will just panic. So might as well `log.Fatal` here and simplify the code both here and in `main`.

Reply:
Makes sense. We can fail on error with a one-liner func call `failonf(err, pattern, args...)`.